.github/workflows/clarity-js-sdk-pr.yml

```diff
@@ -20,26 +24,30 @@ jobs:
     runs-on: ubuntu-latest
     # This condition can be removed once the main `stacks-blockchain` workflow creates pre-releases
```

Comment:
Thoughts on creating a gh issue to do this and linking it to this comment?

Reply:
i stand corrected - this is still needed, it works different than i had thought. 
the premise here is that we can create a release tag for a release candidate, i.e. `2.4.0.0.1-rc1`, and this workflow should not be run (even though it's set to be triggered since it's a valid tag. 

one more reason it may be useful to move to a pushing  tag vs using an input tag.
an issue should be created